scripts/getDependencies.pl

```diff
@@ -218,14 +218,85 @@
 		sha1 => '174bba1574c37352b0eb2c06e02b6403738ad57c'
 	});
 
-my @dependencies = split(',', $dependencyList);
+my %system_jars = (
+	json_simple => {
+		url => 'https://repo1.maven.org/maven2/com/googlecode/json-simple/json-simple/1.1.1/json-simple-1.1.1.jar',
+		fname => 'json-simple.jar',
+		sha1 => 'c9ad4a0850ab676c5c64461a05ca524cdfff59f1',
+	},
+	jcstress => {
+		url => 'https://builds.shipilev.net/jcstress/jcstress-tests-all-20240222.jar',
+		fname => 'jcstress-tests-all-20240222.jar',
+		sha1 => '200da75e67689e8a604ec6fe9a6f55b2c000b6ce',
+	},
+	ant_launcher => {
+		url => 'https://repo1.maven.org/maven2/org/apache/ant/ant-launcher/1.8.1/ant-launcher-1.8.1.jar',
+		dir => 'apache-ant/lib',
+		fname => 'ant-launcher.jar',
+		is_system_test => 1
+	},
+	asm => {
+		url => 'https://repository.ow2.org/nexus/content/repositories/releases/org/ow2/asm/asm/9.0/asm-9.0.jar',
+		dir => 'asm',
+		fname => 'asm.jar',
+		is_system_test => 1
+	},
+	cvsclient => {
+		url => 'https://repo1.maven.org/maven2/org/netbeans/lib/cvsclient/20060125/cvsclient-20060125.jar',
+		dir => 'cvsclient',
+		fname => 'org-netbeans-lib-cvsclient.jar',
+		is_system_test => 1
+	},
+	hamcrest_core => {
+		url => 'https://repo1.maven.org/maven2/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar',
+		dir => 'junit',
+		fname => 'hamcrest-core.jar',
+		is_system_test => 1
+	},
+	junit => {
+		url => 'https://repo1.maven.org/maven2/junit/junit/4.12/junit-4.12.jar',
+		dir => 'junit',
+		fname => 'junit.jar',
+		is_system_test => 1
+	},
+	log4j_api => {
+		url => 'https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-api/2.15.0/log4j-api-2.15.0.jar',
+		dir => 'log4j',
+		fname => 'log4j-api.jar',
+		is_system_test => 1
+	},
+	log4j_core => {
+		url => 'https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-core/2.15.0/log4j-core-2.15.0.jar',
+		dir => 'log4j',
+		fname => 'log4j-core.jar',
+		is_system_test => 1
+	},
+	mauve => {
+		url => 'https://ci.adoptium.net/job/systemtest.getDependency/lastSuccessfulBuild/artifact/systemtest_prereqs/mauve/mauve.jar',
+		dir => 'mauve',
+		fname => 'mauve.jar',
+		is_system_test => 1
+	},
+	tools => {
+		url => 'https://ci.adoptium.net/job/systemtest.getDependency/lastSuccessfulBuild/artifact/systemtest_prereqs/tools/tools.jar',
+		dir => 'tools',
+		fname => 'tools.jar',
+		is_system_test => 1
+	});
 
+my %jars_to_use;
+if ($path =~ /system_lib/) {
+	%jars_to_use = (%system_jars);
+} else {
+	%jars_to_use = %base;
```

Comment:
Is bracket still needed for `%system_jars` if we don't append it with another arrays?

Reply:
Yes, Since`%system_jars` is being assigned directly to `%jars_to_use` and  there is no additional operation happening, the brackets are not strictly necessary.